Controllers/AccessesController.cs

```diff
@@ -0,0 +1,246 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using StoresManagement.Constants;
+using StoresManagement.Data;
+using StoresManagement.Models;
+using StoresManagement.ViewModels;
+
+namespace StoresManagement.Controllers
+{
+    [Authorize(Roles = "Manager")]
```

Comment:
could you make use of StoresManagement.Constants.UserRoles type, as I guess it was reason why you introduced it. Please look through everywhere where you use hardcoded value like "Manager", "Administrator", "Seller",  "Inactive" and use UserRoles.Manager,..., UserRoles.Seller respectively

Reply:
Done. 👍